README.md

```diff
@@ -51,3 +51,20 @@ Travis builds the entire universe represented by this Opam repository.
 [OCaml]:  http:/ocaml.org
 [Travis]: https://travis-ci.org/xapi-project/xs-opam
 
+### Using it on your travis builds
+
+Load the config on the install step for .travis-docker.sh at install time, then run the script for testing your package:
+```
+language: c
+sudo: required
+service: docker
+install:
+  - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
+  - wget https://raw.githubusercontent.com/psafont/xs-opam/central-config/tools/xs-opam-ci.env
```

Comment:
The `psafont` should be `xapi-project`?

Reply:
I thought I had changed it before pushing it, thanks